docs/pages/faq/react-vs-react-native.md

```diff
@@ -0,0 +1,61 @@
+---
+title: React vs React Native
+---
+
+On the surface, there's very little difference between React and React Native- that's one of the great parts about React Native (RN for short). If you're a web developer and have used React, good news- you're (almost) a React Native pro! That being said, React and RN are separate and distinct tools, and like all great tools, they've each been built for a specific purpose in mind. Let's go over some of their similarities, differences, and where each of them shines brightest.
```

Comment:
I would change: "Let's go over some of their similarities, differences, and where each of them shines brightest."

To this on its own line so it's super clear what the reader gets from the content:
Below we cover their similarities, differences, and in what situations it makes sense to use React vs. React Native.